app/src/sharedTest/java/org/oppia/app/testing/TopicTestActivityForStoryTest.kt

```diff
@@ -0,0 +1,68 @@
+package org.oppia.app.testing
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers.containsString
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+import org.oppia.app.topic.TopicTab
+import org.oppia.app.utility.EspressoTestsMatchers
+
+/** Tests for [TopicTestActivityForStory]. */
+@RunWith(AndroidJUnit4::class)
+class TopicTestActivityForStoryTest {
+
+  @Test
+  fun testTopicTestActivityForStory_defaultTabIsPlay_isSuccessful() {
+    launch(TopicTestActivityForStory::class.java).use {
+      onView(withId(R.id.topic_tabs_container)).check(
+        matches(
+          EspressoTestsMatchers.matchCurrentTabTitle(
```

Comment:
Remove `EspressoTestsMatchers` and use `matchCurrentTabTitle` from import.

Reply:
Done